src/components-examples/material/card/card-media-size/card-media-size-example.html

```diff
@@ -17,7 +17,7 @@
     <mat-card-title-group>
       <mat-card-title>Shiba Inu</mat-card-title>
       <mat-card-subtitle>Medium</mat-card-subtitle>
-      <img mat-card-md-image src="https://material.angular.io/assets/img/examples/shiba2.jpg" >
+      <img mat-card-md-image src="https://material.angular.io/assets/img/examples/shiba2.jpg"  alt='example2'>
```

Comment:
This isn't a useful `alt` description. Maybe it should say something like "Image of a Shiba Inu"? Also the attribute should use single quotes.